dataframe/dataframe.go

```diff
@@ -1169,17 +1203,17 @@ func (df DataFrame) InnerJoin(b DataFrame, keys ...string) DataFrame {
 	for _, key := range keys {
 		i := df.colIndex(key)
 		if i < 0 {
-			errorArr = append(errorArr, fmt.Sprintf("can't find key %q on left DataFrame", key))
+			errorArr = append(errorArr, createErr("df.InnerJoin()", "can't find key %q on left DataFrame", key).Error())
```

Comment:
Here you do the same thing... You want a `string`, so why first make a wrapped error?